hack/util.sh

```diff
@@ -804,7 +811,7 @@ os::util::get_object_assert() {
   local request=$2
   local expected=$3
 
-  res=$(eval oc get $object -o go-template=\"$request\")
+  res=$(eval oc get "${object}" -o go-template="\"${request}\"")
```

Comment:
That's a good thing - I'm working to remove this and `tryuntil` as soon as I can.